lib/generators/react_on_rails/hello_world_example_generator.rb

```diff
@@ -0,0 +1,46 @@
+require "rails/generators"
+
+module ReactOnRails
+  module Generators
+    class HelloWorldExampleGenerator < Rails::Generators::Base
+      hide!
+      source_root File.expand_path("../templates", __FILE__)
+
+      # --with-redux
+      class_option :with_redux,
+                   type: :boolean,
+                   default: false,
+                   description: "Include Redux package"
+      # --with-server-rendering
+      class_option :with_server_rendering,
+                   type: :boolean,
+                   default: false,
+                   description: "Enable server-rendering"
+
+      def insert_hello_world_route
+        route "get 'hello_world', to: 'pages#index'"
```

Comment:
let's use "hello_world#index".

pages is a common for a controller name so likely for a conflict